### PR TITLE
Prepare for release of WildFly 26.1.0 (EE 8 Full Platform + Web Profile on Java SE 8)

### PR DESCRIPTION
**Pull request by scottmarlow**
Signed-off-by: Scott Marlow <smarlow@redhat.com>

### CODE REVIEW

**Comment by scottmarlow** [comment]
Will merge this tomorrow.

File: WildFly_26.1.0.Final/jakarta-web-profile.adoc [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+----
+
+Jakarta Contexts and Dependency Injection 2.0 TCK
+
+Download URL & SHA-256
+https://download.eclipse.org/jakartaee/cdi/2.0/cdi-tck-2.0.6-dist.zip[cdi-tck-2.0.6-dist.zip, 75e969a7a3b3c77332154a2008309aad821a923d8684139242048a7640762808 ]
+
+TCK result summary:
+----
+Tests run: 1807, Failures: 0, Errors: 0, Skipped: 0
```
Comment (bstansberry) [inline WildFly_26.1.0.Final/jakarta-web-profile.adoc:234]: @scottmarlow I believe the next line is missing. Should be:

----

I was looking at another one of these earlier today and the formatting toward the bottom looked off; I suspect this is why.

File: WildFly_26.1.0.Final/jakarta-full-platform.adoc [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+----
+
+Jakarta Contexts and Dependency Injection 2.0 TCK
+
+Download URL & SHA-256
+https://download.eclipse.org/jakartaee/cdi/2.0/cdi-tck-2.0.6-dist.zip[cdi-tck-2.0.6-dist.zip, 75e969a7a3b3c77332154a2008309aad821a923d8684139242048a7640762808 ]
+
+TCK result summary:
+----
+Tests run: 1807, Failures: 0, Errors: 0, Skipped: 0
```
Comment (bstansberry) [inline WildFly_26.1.0.Final/jakarta-full-platform.adoc:617]: Same comment here; needs a line with '----'

Comment (scottmarlow) [reply WildFly_26.1.0.Final/jakarta-full-platform.adoc:617]: Thanks for noticing!  Fixed!

File: WildFly_26.1.0.Final/jakarta-web-profile.adoc (hunk 1 above)
Comment (scottmarlow) [reply WildFly_26.1.0.Final/jakarta-web-profile.adoc:234]: Fixed!

**Pull request merged**